libraries/lvm.rb

```diff
@@ -19,7 +19,7 @@ module LVMCookbook
   def require_lvm_gems
```

Comment:
Perhaps a better solution to this would be what @coderanger suggested, just adding a return if the LVM module is already defined to the top of this method.

ie.
```
def require_lvm_gems
  return if defined?(LVM)
  ...
  if node['lvm']['cleanup_old_gems']
```

This would actually speed up the method even further.